files/install_config_agent_yum.sh

```diff
@@ -14,6 +14,9 @@ set -x
 #connect eth1 and use dhcp to get address from neutron network
 nmcli d connect eth1
 
+#set UK timezone
+ln -sf /usr/share/zoneinfo/Europe/London /etc/localtime
```

Comment:
Can we change this to use UTC time instead of localtime? Personal preference but it makes consistency easier and removes any complications around BST.